credentials/alts/internal/handshaker/handshaker.go

```diff
@@ -298,11 +298,11 @@ func (h *altsHandshaker) doHandshake(req *altspb.HandshakerReq) (net.Conn, *alts
 
 func (h *altsHandshaker) accessHandshakerService(req *altspb.HandshakerReq) (*altspb.HandshakerResp, error) {
 	if err := h.stream.Send(req); err != nil {
-		return nil, err
+		return nil, fmt.Errorf("failed to send handshaker request: %w", err)
```

Comment:
Can we include ALTS in the error message, as handshaker might be too generic? For example, "failed to send ALTS handshaker request"? Similarly for the response.

Reply:
Good point. Done.